bin/packages/build.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * script to build WordPress packages into `build/` directory.
+ *
+ * Example:
+ *  node ./scripts/build.js
+ */
+
+/**
+ * External Dependenceis
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const glob = require( 'glob' );
+const babel = require( 'babel-core' );
+const chalk = require( 'chalk' );
+const mkdirp = require( 'mkdirp' );
+
+/**
+ * Internal dependencies
+ */
+const getPackages = require( './get-packages' );
+
+/**
+ * Module Constants
+ */
+const PACKAGES_DIR = path.resolve( __dirname, '../../packages' );
+const SRC_DIR = 'src';
+const BUILD_DIR = {
+	main: 'build',
+	module: 'build-module',
+};
+const DONE = chalk.reset.inverse.bold.green( ' DONE ' );
+
+/**
+ * Babel Configuration
+ */
+const babelDefaultConfig = require( '@wordpress/babel-preset-default' );
```

Comment:
We should load setup from `package.json` - we use some overrides.

Reply:
Yes, this is also what I figured out when trying to update this config yesterday.

I published new version of Babel preset to include missing async generator functions plugin.

Reply:
#6893 will cover the support for async generator functions.